packages/stargate/src/stargateclient.ts

```diff
@@ -268,7 +279,61 @@ export class StargateClient {
     if (this.tmClient) this.tmClient.disconnect();
   }
 
-  public async broadcastTx(tx: Uint8Array): Promise<BroadcastTxResponse> {
+  public async broadcastTx(
+    tx: Uint8Array,
+    timeoutMs = 60_000,
+    pollIntervalMs = 3_000,
+  ): Promise<BroadcastTxResponse> {
+    const txId = toHex(sha256(tx));
```

Comment:
Can make this upper case? This is the typical representation in Cosmos (see https://cosmos.bigdipper.live/transactions)